### PR TITLE
Enable autofocus directive in angular

### LINKED ISSUE #1880: enable autofocus attribute for all form elements
The autofocus attribute does not work correctly in forms loaded via angular.
In #205 this was fixed only for the login screen.
we could add the `focus="True"` attribute to the other input elements but since there are already a few (currently ineffective) `autofocus` attributes in the views, it would be better to overwrite the `autofocus` directive in angular.

### PR DESCRIPTION
**Pull request by plettich**
Make the autofocus attribute work when pages were loaded from angular.
Added and fixed some uses of the `autofocus` attribute.

Fixes #1880

### CODE REVIEW

**Comment by plettich** [comment]
based on https://gist.github.com/kirkstrobeck/599664399dbc23968741

File: privacyidea/static/components/login/views/login.html [hunk 1]
```diff
@@ -56,7 +56,8 @@ <h2 ng-show="piLoginText" class="form-signin-heading">{{ piLoginText }}</h2>
                        class="form-control round-top"
                        ng-model="login.username" required
                        placeholder="{{ 'Username'|translate }}"
-                       focus="true"/>
+                       focus="true"
```
Comment (NuvandaPV) [inline privacyidea/static/components/login/views/login.html]: Does the `[focus]` here still do anything? If I read correctly, the focus-directive is removed, so this should probably be removed as well.

Comment (plettich) [reply privacyidea/static/components/login/views/login.html]: Fixed in 17c361fc

**Comment by NuvandaPV** [comment]
Fixes #1880

**Pull request merged**